src/Components/CustomElements/src/PublicAPI.Shipped.txt

```diff
@@ -1 +1 @@
-﻿#nullable enable
+#nullable enable#nullable enableMicrosoft.AspNetCore.Components.Web.CustomElementsJSComponentConfigurationExtensionsstatic Microsoft.AspNetCore.Components.Web.CustomElementsJSComponentConfigurationExtensions.RegisterCustomElement<TComponent>(this Microsoft.AspNetCore.Components.Web.IJSComponentConfiguration! configuration, string! identifier) -> void
```

Comment:
Corruption here or just line endings the script didn't know it needed to handle❔

Reply:
I'm guessing it's the latter, might need to update the script.